djangobnb_backend/property/tasks.py

```diff
@@ -0,0 +1,58 @@
+from django.core.mail import send_mail, EmailMultiAlternatives, send_mass_mail
+from django.template.loader import render_to_string
+from django.utils.html import strip_tags
+from celery import shared_task
+from .models import Reservation
+from django.conf import settings
+
+
+@shared_task
+def booking_confirmed(reservation_id):
+    reservation = Reservation.objects.get(id=reservation_id)
+    property_owner = reservation.property.host
+
+    context = {
```

Comment:
Email building part could be separated, this and following lines would be better a function i.e fucntion like ```def generate_email_content() -> text content```